cms/static/cms/sass/mixins/_buttons.scss

```diff
@@ -68,4 +68,7 @@
             }
         }
     }
+    &.cms-btn-disabled {
+        color: lighten($color, 50%);
```

Comment:
@FinalAngel can you move it up to line 63? this one doesn't cover all cases and doesn't have the important function